server/downloadUrl.js

```diff
@@ -1,8 +1,10 @@
 var electronSettings = Meteor.settings.electron || {};
 var latestVersion = electronSettings.version;
-var downloadUrl = electronSettings.downloadUrl;
-if (downloadUrl) {
-  downloadUrl = downloadUrl.replace('{{version}}', latestVersion);
+if (electronSettings.downloadUrl){
+  var downloadUrls = {
+    "win32": electronSettings.downloadUrl.win32.replace('{{version}}', latestVersion),
```

Comment:
We need to check if `downloadUrl` contains `win32` and `osx`, otherwise these lines might throw